docs/sql/reference/select.txt

```diff
@@ -12,7 +12,7 @@ Synopsis
 ::
 
     SELECT [ ALL | DISTINCT ] * | expression [ [ AS ] output_name ] [, ...]
-      FROM table_ident [ [AS] table_alias ]
+      FROM table_reference [{CROSS JOIN | , } table_reference [{CROSS JOIN | ,} ...]]
```

Comment:
pls rewrite FROM synopsis as ``relation``

and split it up in a subrule to: ``table_reference | joined_relation``

where joined_relation:

``relation join_operator relation``

``join_operator`` then is ``, | CROSS JOIN``

this is more readable and future-proof